client/src/components/Map.js

```diff
@@ -22,7 +22,7 @@ class Map extends PureComponent {
     if (!data.length) {
       return null
     }
-    const bounds = latLngBounds([data[0].lat, data[0].lng], [data[1].lat, data[1].lng])
+    const bounds = data[1] ? latLngBounds([data[0].lat, data[0].lng], [data[1].lat, data[1].lng]) : null
```

Comment:
to avoid map crashing

Reply:
Looks like this could be more readable / requires a comment in the code as well.
Another option is a tiny function "hasData" that would do this check and improve readability.

Reply:
@yanivmichaelis Agree, I will change this